File: Java/LinearSearch.java
public class LinearSearch {
    public static int linearSearch(int[] array, int num){
        for(int i = 0; i < array.length; i++){
            if(array[i] == num){
                return i;
            }
        }
        return -1;
    }

    public static void main(String[] args){
        int[] array = new int[1000000];
        for(int i = 0; i < 1000000; i++){
            array[i] = i;
        }
        long startTime = System.currentTimeMillis();
        System.out.println(linearSearch(array, 1000000));
        long endTime = System.currentTimeMillis();
        System.out.println("The execution of the algorithm took: " + (endTime - startTime) + " milliseconds");
    }
}

File: Python/LinearSearch.py
import time

def linearSearch(array, number: int) -> int:
    for i in range(len(array)):
        if array[i] == number:
            return i
    
    return -1

array = [i for i in range(1000000)]

startTime = time.time()
print(linearSearch(array, 1000000))
endTime = time.time()
print(f"The execution of the algorithm took: {(endTime - startTime)*1000} milliseconds")